src/content/docs/en/guides/rss.mdx

```diff
@@ -66,9 +67,11 @@ The package [`@astrojs/rss`](https://github.com/withastro/astro/tree/main/packag
     ```
 </Steps>
 
+<ReadMore>See the [`@astrojs/rss` README](https://github.com/withastro/astro/tree/main/packages/astro-rss) for the full configuration reference.</ReadMore>
```

Comment:
Now I **really** want this turned into an integration, so we just have the standard integration guide page in docs!!

Reply:
Yup, totally agree!